reversion_compare/compare.py

```diff
@@ -122,6 +135,9 @@ def get_many_to_many(self):
         """
         if self.field.get_internal_type() != "ManyToManyField":  # FIXME!
             return ([], [], [], [])  # TODO: refactory that
+        elif self.value is DOES_NOT_EXIST:
+            return ([], [], [], [])
```

Comment:
Ah, now I understand the refactor note. I didn't understand why it was there in the first place.